astrokat/observe_main.py

```diff
@@ -237,6 +238,7 @@ def __init__(self, opts, correlator=None):
 
     def __enter__(self):
         # Verify subarray setup correct for observation before doing any work
+        user_logger.warn('Running astrokat version - {}'.format(katversion.get_version()))
```

Comment:
I think this is better logged as `info` instead of `warn`

So like you were explaining, the log didn't appear when you put it in `__init__`?

Reply:
> I think this is better logged as `info` instead of `warn`. Have a look at this: https://stackoverflow.com/questions/2031163/when-to-use-the-different-log-levels
> 
Strictly speaking, it is not a warning. However, warnings show up as a different colour which makes it more visible.

> So like you were explaining, the log didn't appear when you put it in `__init__`?
No, it only seems to work in __enter__ for dryrun. It has something to do with how katcorelib initialises logger.

Reply:
I agree with @samtwum that this should be info level, not warning.  It isn't special.  In normal operation the user shouldn't care much about the version - it is just there for auditing later.